spec/index.bs

```diff
@@ -8410,12 +8410,9 @@ dictionary GPUFragmentState
                     - |entryPoint| must have a [=shader stage output=] with [=location=] equal to |index|
                          and [=blend_src=] omitted or equal to 0.
             - If |usesDualSourceBlending| is `true`:
-                - All the [=shader stage output=] values of |entryPoint| must have a [=blend_src=] attribute.
```

Comment:
Shouldn't we keep this to requires that there is `@location(0) @blend_src(0)` and not just `@location(0)`?

Reply:
It is because in [WGSL SPEC](https://gpuweb.github.io/gpuweb/wgsl/#input-output-locations) we require
`
If any entry in members specifies a blend_src attribute， members must contain exactly 2 entries, one with @location(0) @blend_src(0) and one with @location(0) @blend_src(1).
`
So I think it is enough to require there is at least `@location(0) @blend_src(1)`.

Reply:
SGTM